integrations/server/test_covidcast_meta.py

```diff
@@ -69,6 +71,7 @@ def test_round_trip(self):
               for gv, v in zip(('geo1', 'geo2'), (10, 20)):
                 self.cur.execute(template % (src, sig, tt, gt, tv, gv, v))
     self.cnx.commit()
+    update_cache(args=None)
```

Comment:
Will this try to connect to the production database? I don't see any provision in `update_cache` for connecting to a mock database; regardless of what connection the test makes, `update_cache` uses the database details in the secrets file.

Unless the test environment just has identical secrets but a different database?

Reply:
My understanding is that the test environment is isolated from the outside internet via the Docker container, and could not contact the production database even if it wanted to. 

That said, we can be confident that this particular test writes to the `covidcast` table on host `delphi_web_epidata`, which is the Docker container. The test then asks for the metadata from host `delphi_web_epidata`, and receives the expected result. So *someone* must be writing to the `covidcast_meta_cache` table on host `delphi_web_epidata`, so if it's not `update_cache` using a secrets file modified by the testing machinery to point to `delphi_web_epidata` instead of the production server, then we have a whole nother set of problems.